### PR TITLE
Add BAKER3

### PR DESCRIPTION
**Pull request by rlibouba**
Hi, 
I opened this pull request to add BRAKER3. I have specified the use of singularity to launch BRAKER3. I'm not sure of the parameters. 
For the mem and cores parameters, I have entered what we have set on usegalaxy.fr.

Thank you in advance for your help.
Have a nice day!

### CODE REVIEW

File: files/galaxy/tpv/tools.yml [hunk 1]
```diff
@@ -1006,6 +1006,15 @@ tools:
       require:
         - singularity
 
+  toolshed.g2.bx.psu.edu/repos/galaxy-tools/braker3/braker3/.*:
+    mem: 4
+    cores: 6
+    params:
+      singularity_container_id_override: docker://teambraker/braker3:v.1.0.4
```
Comment (bgruening) [inline files/galaxy/tpv/tools.yml:1013]: @rlibouba I don't know if it works with the `docker:` prefix. Is that what is used in TPV usegalaxy.fr as well?

We can merge and test I think.

Comment (abretaud) [reply files/galaxy/tpv/tools.yml:1013]: we use the docker:// with the old sorting hat, it works fine

Comment (rlibouba) [reply files/galaxy/tpv/tools.yml:1013]: I tested BRAKER3 on UseGalaxy.eu, and the tests failed. The error reads "/gmes/gmes_petap.pl: No such file or directory".

Comment (abretaud) [reply files/galaxy/tpv/tools.yml:1013]: oh maybe it's not using the singularity image?

Comment (bgruening) [reply files/galaxy/tpv/tools.yml:1013]: I think its the wrong tool ID

**Pull request merged**